### PR TITLE
Translations update from MoJ Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [MoJ Weblate](https://moj.weblate.cloud) for [opg-modernising-lpa/online-content](https://moj.weblate.cloud/projects/opg-modernising-lpa/opg-modernising-lpa/).

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by nia-campbell — APPROVED** [review]
Updated Welsh content in line with English changes, and checked apostrophes!

File: lang/cy.json [hunk 1]
```diff
@@ -296,14 +296,14 @@
...
+    "weWillContactYourCertificateProvider": "Byddwn yn cysylltu â {{.FirstNames}}, ac yn gofyn iddynt gadarnhau eu hunaniaeth a darparu’r dystysgrif ar gyfer eich LPA.",
     "opgWillContactYourCertificateProvider": "Bydd Swyddfa’r Gwarcheidwad Cyhoeddus yn cysylltu â {{.FullName}} ac yn gofyn iddynt gadarnhau pwy ydynt a darparu’r dystysgrif ar gyfer eich LPA.",
     "weWillThenContactAttorneys": {
         "one": "Byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrnai.",
         "two": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
         "few": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
         "many": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
-        "other": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod."
+        "other": "Byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod."
```
Comment (hawx) [inline lang/cy.json:306]: should this be for all variations?

**Review by andrewpearce-digital — DISMISSED** [review]

File: lang/cy.json [hunk 2]
```diff
@@ -554,7 +554,7 @@
         "many": "Enwau’r atwrneiod wrth gefn",
         "other": "Enwau’r atwrneiod wrth gefn"
     },
-    "readDonorNameLpa": "Darllen LPA {{ .DonorFullName }}",
+    "readDonorNameLpa": "Darllen LPA {{possessive .DonorFullName}}",
```
Comment (hawx) [inline lang/cy.json:557]: `possessive` doesn't exist for the Welsh translations, do you need it to work here with English style rules?

File: lang/cy.json (hunk 1 above)
Comment (nia-campbell) [reply lang/cy.json:306]: I think so - it's removing the word 'Then' from the start of the sentence. 

I can only see .one and .other in Weblate though. Are you able to change these your side, @hawx ?

File: lang/cy.json (hunk 2 above)
Comment (nia-campbell) [reply lang/cy.json:557]: Nope, sorry I thought it needed to have the same bracketed text. It's fine without.

**Review by hawx — APPROVED** [review]

**Review by hawx — DISMISSED** [review]

**Review by andrewpearce-digital — DISMISSED** [review]

**Review by andrewpearce-digital — DISMISSED** [review]

**Review by hawx — APPROVED** [review]

**Pull request merged**